src/Rendering/babylon.geometryRenderer.ts

```diff
@@ -0,0 +1,164 @@
+module BABYLON {
+    export class GeometryRenderer {
+        private _scene: Scene;

```

Comment:
The thing is "MultiRenderTargetRenderer" is not precise enough.
This renderer actually renders the geometry buffer (sometimes called g-buffer) which contains depths, normals, and in the future probably albedo and some lightning information.
It uses MRT for this purpose, but it is not the end goal.